packages/discord.js/src/structures/Poll.js

```diff
@@ -93,6 +137,33 @@ class Poll extends Base {
     return new Date(this.expiresTimestamp);
   }
 
+  /**
+   * Whether or not this poll is a partial
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return (
+      typeof this.question.text !== 'string' ||
+      typeof this.layoutType !== 'number' ||
+      typeof this.allowMultiselect !== 'boolean' ||
+      this._partial
```

Comment:
I removed the `typeof this.question.text` check since it needs to be populated always (even if it is partial, where `...question.text` is populated with `''`), for compatibility reasons.

The other typeof checks however are not nullable on the API, therefore it would make the poll partial if they aren't there, and I can't supply it in the action without making the data inaccurate.

By default, `this._partial` will always be true if the poll was created as a partial poll in the action. I did move `this._partial` to the top of the checks though